handsontable/src/renderers/textRenderer/textRenderer.js

```diff
@@ -44,6 +44,9 @@ export function textRenderer(instance, TD, row, col, prop, value, cellProperties
     // this is faster than innerHTML. See: https://github.com/handsontable/handsontable/wiki/JavaScript-&-DOM-performance-tips
     fastInnerText(TD, escaped);
   }
+  if (cellProperties.type === 'time') {
```

Comment:
The same case is mentioned here https://github.com/handsontable/handsontable/pull/8906#discussion_r763077563